src/network/os1/server/codec/game/PlaySoundEncoder.ts

```diff
@@ -0,0 +1,15 @@
+import type Packet from '#/io/Packet.ts';
+
+import MessageEncoder from '#/network/server/codec/MessageEncoder.ts';
+import type PlaySound from '#/network/server/model/game/PlaySound.ts';
+
+export default class PlaySoundEncoder extends MessageEncoder {
```

Comment:
PlaySound is called SYNTH_SOUND on their end, also "immediate" may be "loops" if it's like older clients. Not every synth can loop (part of its data to say what period of time), so it can appear to do nothing.